app/views/manage_assessments/assignments/new.html.erb

```diff
@@ -1,5 +1,45 @@
-<%= simple_form_for @assignment, url: manage_assessments_outcome_coverage_assignments_path do |form| %>
-  <%= form.input :name, label: t(".assignment") %>
-  <%= form.input :problem, label: t(".problem") %>
+<%= content_for :full_bleed do %>
+  <div class="madlib-header">
+    <h1 class="headline-narrow">
+      Add assignment
+    </h1>
+    <h2 class="headline-narrow-subhead">
+      <%= @assignment.outcome_coverage.coverage.course.number %>
```

Comment:
We would need a string of associations and delegates such that `assignment.course_number` works. 

something like:

```ruby
class Assignment
  has_one :coverage, through: :outcome_coverage
  delegate :course_number, to: :coverage
end

class Coverage
  delegate :number, to: :course, prefix: true
end
```

... should work.